gef.py

```diff
@@ -5373,7 +5363,7 @@ def reset():
        "True" if verbose else "False",
        current_arch.syscall_register,
        cs_arch, cs_mode,
-       16 if is_elf64() else 8,
+       get_memory_alignment() * 2,
```

Comment:
I tend to prefer `current_arch.ptrsize` to `get_memory_alignment()`, more explicit.